oligo_designer_toolsuite/oligo_selection/_generate_oligosets.py

```diff
@@ -52,7 +53,7 @@ def __init__(
         min_oligoset_size: int,
         oligos_scoring: OligoScoringBase,
         set_scoring: SetScoringBase,
-        heuristic_selection: Callable = None,
+        heuristic_selection: OligoSelectionPolicy,
         distance_between_oligos: int = 0,
         max_oligos: int = 5000,
```

Comment:
Hmm, good point. The logic is to be conservative: we limit so that in the worst case (GraphBased, which is the default approach) doesn't crash.. What do you think?